lib/src/arbiter.dart

```diff
@@ -60,3 +60,57 @@ class PriorityArbiter extends Arbiter {
     ]);
   }
 }
+
+/// Arbiter Roun Robin.
+class RoundRobinArbiterlogic extends Arbiter {
+  ///Mask to define pending requests to be attended
+  ///i.e. If bit 4 is granted in a 8 bit request, [requestMask] will be 11110000
+  ///turning off the bit granted along with all previous bits
+  List<Logic> requestMask = [];
```

Comment:
We should leave internal implementation details as private by starting with a `_`, so `_requestMask` instead of `requestMask`.  Same for `_grantMask`.